src/content/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder.mdx

```diff
@@ -130,3 +134,4 @@ Once you have built your chart you can:
 * [Change the type of chart](/docs/chart-builder/use-chart-builder/use-charts/use-your-charts#change-appearance). Based on the data you specified, the query builder selects the chart type that displays your results most effectively. However, you can choose from other [available chart types](/docs/insights/use-insights-ui/manage-dashboards/insights-chart-types) to present your data in the visual format that you want to use.
 * [Share your chart](/docs/chart-builder/use-chart-builder/use-charts/use-your-charts#share-charts).
 * [Add your chart to a dashboard](/docs/chart-builder/use-chart-builder/use-charts/use-your-charts#add-to-dashboard).
+* [Create an NRQL alert condition from your chart](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create-chart).
```

Comment:
I prefer "a NRQL" because I pronounce it "nurkle" in my mind. 

"a NRQL" is consistent with what I'm seeing in this doc: https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create

Reply:
True! I was taken away by how similar it is to SQL.